scripts/author.md

```diff
@@ -0,0 +1,11 @@
+type: page
+title: "关于 ${name}"
+author: ${name}
+
+---
+
+## ${name}
+
+${intro}
+
+NOTE: this an auto generated page
```

Comment:
I meant to place it here, it's the author's duty to remove it before push it to remote and also the reviewers' duty to pick it out. Though we should actually have git hook for pre-commit and abort the commit if we find certain keywords like `FIXME`, `TODO`, an example would be influxdb https://github.com/influxdata/influxdb/blob/master/.hooks/pre-commit#L16  I was hopping all those boilerplate things like git hook, attributes, editorconfig can be done using Ayi's generator feature (currently, I am just copying those stuff from Ayi's repo), but I guess it would take a while since I am the only one working on it and I got xephon-b to take care of  =w=